appveyor.yml

```diff
@@ -66,6 +66,10 @@ install:
     - cmd: conda config --add channels conda-forge
     - cmd: conda info
     - cmd: conda install -n root --quiet --yes conda-build anaconda-client jinja2 setuptools
+    # Workaround for Python 3.4 and x64 bug in latest conda-build.
+    # FIXME: Remove once there is a release that fixes the upstream issue
+    # ( https://github.com/conda/conda-build/issues/895 ).
+    - cmd: if "%TARGET_ARCH%" == "x64" if "%CONDA_PY%" == "34" conda install conda-build=1.20.0 --yes
```

Comment:
If this gets added back, `curl` will fail to build on Windows Python 3.4 64-bit. We need a later version of `conda-build` to fix the build here and `1.20.0` simply isn't recent enough.